providers/threeplay.go

```diff
@@ -77,8 +79,9 @@ func (c *ThreePlayProvider) GetProviderJob(id string) (*database.ProviderJob, er
 // when the media_file_url param is provided
 func (c *ThreePlayProvider) DispatchJob(job *database.Job) error {
 	jobLogger := c.logger.WithFields(log.Fields{"JobID": job.ID, "Provider": job.Provider})
+	callParams := threeplay.CallParams{APIKey: c.config.APIKeyByJobType[job.JobType]}
 	// Review job route
-	if mediaFileID, ok := job.ProviderParams["media_file_id"]; ok {
+	if providerID, ok := job.ProviderParams["provider_id"]; ok {
```

Comment:
Should this be ``"ProviderID"`` or do we actually support both forms? If the former, is it another reason to make it a method to avoid the mistake in the future? :grin: If do support both forms, can we not? :see_no_evil:

Reply:
This is a weird case. I see your point. The `provider_id` is a param that the caller passes that references the 3play job they want to generate a review link for. `ProviderID` is what the captions api uses to keep track of the provider IDs returned by the provider.
In this case, they're the same and both get stored because the naming convention that was chosen, for some reason, in the api is to use capitalized key fields.
I agree that we should probably have a helper method (I'll add it), but in this case, because we automatically store all the providerParams when creating the job, we'd have to store both.
I guess a solution to that confusion would be to maybe rename it to `transcript_id`?

Reply:
Ohh I see. Yeah using a name that doesn't cause confusion may be better.

Would we be able to abstract it in the helper method? (like is there some sort of preference between those 2 or are they completely unrelated?) If the helper method is the only one dealing with both ``"ProviderID"`` and ``"provider_id"``, maybe we can keep it with an inline comment.